tools/indexers/trials.js

```diff
@@ -332,11 +332,16 @@ const trialMapping = {
     },
     discrepancies: {
       properties: {
-        public_title: getDiscrepancyRecordMapping({
```

Comment:
Agreed. Looks like I missed the mappings here when changed the discrepancy fields :+1: